keyboards/bastardkb/charybdis/3x5/config.h

```diff
@@ -18,55 +18,25 @@
 
 #pragma once
 
-#define VENDOR_ID 0xA8F8
-#define PRODUCT_ID 0x1832
-#define DEVICE_VER 0x0001
-#define MANUFACTURER Bastard Keyboards
-#define PRODUCT Charybdis Nano
+#include "config_common.h"
 
 /* Key matrix configuration. */
-
-// Rows are doubled-up.
-#define MATRIX_ROWS 8
+#define MATRIX_ROWS 8 // Rows are doubled-up.
 #define MATRIX_COLS 5
 
-// Wiring configuration for each half.
-#define MATRIX_ROW_PINS \
-    { B7, C6, D4, B5 }
-#define MATRIX_COL_PINS \
-    { C7, F0, D7, E6, B4 }
-
-#define MATRIX_ROW_PINS_RIGHT \
-    { F0, C6, D4, B5 }
-#define MATRIX_COL_PINS_RIGHT \
-    { C7, B7, D7, E6, B4 }
-
 #define DIODE_DIRECTION ROW2COL
 
-/* Handedness. */
-#define MASTER_RIGHT
-
-/* Bootmagic Lite configuration. */
-#define BOOTMAGIC_LITE_ROW 0
-#define BOOTMAGIC_LITE_COLUMN 0
-#define BOOTMAGIC_LITE_ROW_RIGHT 4
-#define BOOTMAGIC_LITE_COLUMN_RIGHT 0
-
-/* serial.c configuration (for split keyboard) */
-#define SOFT_SERIAL_PIN D2
+/* Disable action features. */
+#define NO_ACTION_MACRO    // Disable old-style macro handling.
+#define NO_ACTION_FUNCTION // Disable old-style function handling.
```

Comment:
These no longer exist, and can be completely removed. 
```suggestion
```

Reply:
Done.